packages/react-components/react-tags/library/src/components/TagGroup/useTagGroup.ts

```diff
@@ -22,12 +23,16 @@ export const useTagGroup_unstable = (props: TagGroupProps, ref: React.Ref<HTMLDi
     appearance = 'filled',
     dismissible = false,
     role = 'toolbar',
+    onTagSelect,
+    ...rest
   } = props;
 
   const innerRef = React.useRef<HTMLElement>();
   const { targetDocument } = useFluent();
   const { findNextFocusable, findPrevFocusable } = useFocusFinders();
 
+  const [items, setItems] = React.useState<Array<TagValue>>([]);
```

Comment:
Why do we control state instead of using `useControllableState`?

Reply:
fixed